modules/RTC/RTC.js

```diff
@@ -340,7 +337,7 @@ export default class RTC extends Listenable {
     _senderVideoConstraintsChanged(senderVideoConstraints) {
         logger.info('Remote max frame height received on bridge channel: ', JSON.stringify(senderVideoConstraints));
         this._senderVideoConstraints = senderVideoConstraints;
-        this.eventEmitter.emit(RTCEvents.SENDER_VIDEO_CONSTRAINTS_CHANGED);
+        this.eventEmitter.emit(RTCEvents.REMOTE_VIDEO_CONSTRAINTS_CHANGED);
```

Comment:
it's a little bit confusing that this emits `REMOTE_VIDEO_CONSTRAINTS_CHANGED` when the "senderVideoConstraintsChanged", especially that `this._senderVideoConstraints` is accessible via `getSenderVideoConstraints`

Reply:
I know, I needed two different events for this, one that gets emitted when the message is received on the bridge channel and the other which notifies JingleSessionPC about this event.
https://github.com/jitsi/lib-jitsi-meet/blob/master/modules/xmpp/JingleSessionPC.js#L587
I went with the message used there. I am open to any others suggestion you might have.